src/config.rs

```diff
@@ -183,6 +201,33 @@ impl Config {
 mod tests {
     use super::*;
 
+    // TODO these are duplicated in render_cmd as well
```

Comment:
I propose to start creating a `tests` module in a follow-up pr to bring the unit tests together.

See the unit tests module example directory tree in https://matklad.github.io/2021/02/27/delete-cargo-integration-tests.html

```
src/
  lib.rs
  tests.rs
  tests/
    foo.rs
    bar.rs
```

@AlexanderThaller what do you think? It seems to be an elegant way to share code between unit tests.

Reply:
Yeah sounds good :D

Reply:
I saw you did the same in [`format_serde_error`](https://github.com/AlexanderThaller/format_serde_error/tree/main/src/test) 😃 

It'll be a new PR then.